pkg/testframework/mock/listers/configmaps.go

```diff
@@ -0,0 +1,22 @@
+package listers
+
+import (
+	coreapi "k8s.io/api/core/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	coreset "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type MockConfigMapNamespaceLister struct {
```

Comment:
doesn't look like this is used anywhere?

Reply:
Looks like that was a leftover from when I tried doing listers for the ClusterVersion and ClusterOperator.  Removed in next push.